tests/benchmarks/FCSBenchmarks/FCSSourceFiles/Program.fs

```diff
@@ -0,0 +1,934 @@
+﻿open System
```

Comment:
This is a file move with the only change being adding extra `..\` due to moving one level deeper.
Rider's Git view shows full history for this file, so I think that means it's not lost.